### PR TITLE
Puts directories above files in the file tree

### PR DESCRIPTION
**Pull request by flukeout**
File tree should look like this now...

### CODE REVIEW

File: src/project/FileTreeView.js [hunk 1]
```diff
@@ -1113,7 +1113,7 @@ define(function (require, exports, module) {
                     isRoot: true,
                     depth: 1,
                     parentPath: this.props.parentPath,
-                    sortDirectoriesFirst: this.props.sortDirectoriesFirst,
+                    sortDirectoriesFirst : this.props.sortDirectoriesFirst,
```
Comment (humphd) [inline src/project/FileTreeView.js]: Revert the change to this file, please

File: src/project/ProjectManager.js [hunk 2]
```diff
@@ -335,7 +335,7 @@ define(function (require, exports, module) {
      * See `ProjectModel.setSortDirectoriesFirst`
      */
     ActionCreator.prototype.setSortDirectoriesFirst = function (sortDirectoriesFirst) {
-        this.model.setSortDirectoriesFirst(sortDirectoriesFirst);
+        this.model.setSortDirectoriesFirst(true);
```
Comment (humphd) [inline src/project/ProjectManager.js]: Should be able to revert this now that the pref will start out at `true`

File: src/project/ProjectManager.js [hunk 3]
```diff
@@ -1255,7 +1255,7 @@ define(function (require, exports, module) {
     CommandManager.register(Strings.CMD_FILE_REFRESH,     Commands.FILE_REFRESH,          refreshFileTree);
 
     // Define the preference to decide how to sort the Project Tree files
-    PreferencesManager.definePreference(SORT_DIRECTORIES_FIRST, "boolean", brackets.platform !== "mac", {
+    PreferencesManager.definePreference(SORT_DIRECTORIES_FIRST, "boolean", true, {
```
Comment (humphd) [inline src/project/ProjectManager.js]: Add a comment above this line:

```
// XXXBramble: we force dirs to get shown at the top
```

**Comment by flukeout** [comment]
@humphd updated with requested changes.
